torch/csrc/jit/frontend/builtin_functions.cpp

```diff
@@ -49,6 +49,12 @@ def ndim(a : Tensor) -> int:
   return a.dim()
 def T(a : Tensor) -> Tensor:
   return a.numpy_T()
+def H(a : Tensor) -> Tensor:
```

Comment:
@SplitInfinity could you help review this?

Reply:
cc. @eellison

Reply:
should we add entries here: https://github.com/pytorch/pytorch/blob/ac99d63f83ceaee4a95e7baa8a52fba09903d00b/torch/csrc/jit/frontend/sugared_value.cpp#L119 ?

Reply:
Looks good!

Reply:
And yes I think you need the changed to `sugared_value.cpp` so that the JIT compiler can recognize these new properties.